doc/sections/install.md

```diff
@@ -1,15 +1,20 @@
-Installation via Opam
----------------------
+Installation via [Opam](http://opam.ocaml.org/)
+-----------------------------------------------
 
-See http://opam.ocaml.org/.
+The version of `lambdapi` that is directly available through Opam (with the
+command `opam install lambdapi`) is currently outdated. We will only publish
+a new version of `lambdapi` Opam pacakge when the development has reached a
+more stable test. For now, we advise you to pin the development repository to
+get the latest bug fixes.
 
 ```bash
-opam install lambdapi
+opam pin add lambdapi https://github.com/Deducteam/lambdapi.git
```

Comment:
please revert, just mention in comment that one can pin add the development version

Reply:
As I said somewhere else, this is better for now.